uc-req.md

```diff
@@ -101,6 +101,25 @@ on "Alice's kitchen TV".
 > ISSUE #40: [Screen availability mechanism for multiple
 > sessions](https://github.com/w3c/presentation-api/issues/40)
 
+### <a name="uc05"></a>UC05: Multiplayer gaming - Pictionary
```

Comment:
Can you provide a link to a description of Pictionary - as I am not sure this game is known globally?

For example, https://en.wikipedia.org/wiki/Pictionary